src/com/epam/javacourse/homework4/Main.java

```diff
@@ -0,0 +1,126 @@
+package com.epam.javacourse.homework4;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        GreenhouseManagement greenhouse = new GreenhouseManagement();
+        Scanner scanner = new Scanner(System.in);
+
+        boolean exit = false;
+        do {
+            System.out.println("Enter 1 to create a new plan, 2 to buy a new plant, 3 to update a plant, 4 to remove a plant, 5 to water the plants, 6 to set the temperature, 7 to display all plants, 8 to exit:");
+            int choice = scanner.nextInt();
+            scanner.nextLine();
+
+
+            if (choice == 1) {
+                System.out.println("Enter the name of the plant:");
+                String name = scanner.nextLine();
+
+                System.out.println("Enter the type of the plant (Flowering Plant, House Plant, or Shrub):");
+                String type = scanner.nextLine();
+
+                System.out.println("Enter the native region of the plant:");
+                String nativeRegion = scanner.nextLine();
+
+                switch (type) {
+                    case "Flowering Plant" -> {
+                        System.out.println("Please enter the flowering time (in days): ");
+                        int floweringTime = scanner.nextInt();
+                        FloweringPlant newPlant = new FloweringPlant(name, type, nativeRegion, floweringTime);
+                        greenhouse.createPlant(newPlant);
```

Comment:
This code does not add the plant to the list as soon as all `createPlant` methods in `GreenHouseManagement` class do not have implementation. You can replace all the `createPlant` calls in this class with `addPlant` to make it work.

Reply:
Done